app/views/users/totp_setup/new.html.erb

```diff
@@ -38,13 +38,7 @@
         <code class="display-block margin-y-2 font-family-mono padding-y-2 padding-x-1 border-base-lighter border-1px text-bold" id="qr-code">
           <%= @code %>
         </code>
-        <%= render ClipboardButtonComponent.new(
-              clipboard_text: @code.upcase,
-              outline: true,
-              class: 'ico ico-copy',
-            ) do %>
-          <%= t('links.copy') %>
-        <% end %>
+        <%= render ClipboardButtonComponent.new(clipboard_text: @code.upcase, outline: true %>
```

Comment:
I think this is gonna be a syntax error for missing a closing paren? 😬 
```suggestion
        <%= render ClipboardButtonComponent.new(clipboard_text: @code.upcase, outline: true) %>
```

Reply:
> I think this is gonna be a syntax error for missing a closing paren? 😬

Whoops! Thanks for catching that.